help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,24 +3,29 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports such authentication methods:
+
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
 To accommodate any enterprise needs, we can develop a customer-specific integration.
 
-All methods can be enabled separately or combined.
+You can enable all methods separately or combined.
```

Comment:
Should it be a separate paragraph? Maybe we can integrate this sentence to the first one?